packages/DropdownMenu/src/Item.styles.js

```diff
@@ -0,0 +1,52 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const itemStyles = css`
+  cursor: pointer;
+  display: block;
+  ${stylers.fontSize()};
+  font-family: ${tokens.fontFamily.default};
+  font-weight: normal;
+  padding: ${tokens.spaceSm} ${stylers.spacer(2)};
+  text-decoration: none;
+  line-height: ${stylers.lineHeight()};
+
+  &:focus {
+    box-shadow: ${tokens.highlight.active.withBorder}
```

Comment:
This is broken.  I regret our naming of this token, but you need to use:
`box-shadow: ${tokens.highlight.active.withBorder.boxShadow};`

😬